File: 30-40 arası örnekler/33.kombinasyon_2.py
import math

print("Kombinasyon(n-r): ")
n=int(input("n: "))
r=int(input("r: "))

f1=math.factorial(n)
f2=math.factorial(r)
f3=math.factorial(n-r)

k=f1/(f2*f3)

print("\nKombinasyon(%d,%d)=%d"%(n,r,k))

File: c/30-40 arası örnekler/33_kombinasyon2.c
#include <stdio.h>
#include <conio.h>

float faktoriyel(int x)
{
    int y;
    float z = 1;
    for (y = 1; y <= x; y++)
        z *= y;
    return z;
}
void main()
{
    int n, r;
    float f1, f2, f3, k;
    printf("Kombinasyon(n-r)\n\n");
    printf("n= ");
    scanf("%u", &n);
    printf("r= ");
    scanf("%u", &r);
    f1 = faktoriyel(n);
    f2 = faktoriyel(r);
    f3 = faktoriyel(n - r);
    k = f1 / (f2 * f3);
    printf("\nKombinasyon(%d,%d)=%0.0f", n, r, k);

    getch();
}
